diff -Nru GXedit1.23/ftp-ls.sh GXedit1.23.new/ftp-ls.sh
--- GXedit1.23/ftp-ls.sh	Tue Feb 23 07:02:43 1999
+++ GXedit1.23.new/ftp-ls.sh	Wed Oct 20 23:26:58 1999
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # ftp.sh output_file host directory user pass
 ftp -n $2 > $1 2> $1 <<EOC
 user $4 $5
diff -Nru GXedit1.23/ftp-put.sh GXedit1.23.new/ftp-put.sh
--- GXedit1.23/ftp-put.sh	Tue Feb 23 07:02:43 1999
+++ GXedit1.23.new/ftp-put.sh	Wed Oct 20 23:27:05 1999
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # ftp-put.sh output_file host directory remote_file local_file user pass
 ftp -n $2 > $1 2> $1 <<EOC
 user $6 $7
diff -Nru GXedit1.23/ftp.sh GXedit1.23.new/ftp.sh
--- GXedit1.23/ftp.sh	Tue Feb 23 07:02:43 1999
+++ GXedit1.23.new/ftp.sh	Wed Oct 20 23:27:11 1999
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # ftp.sh output_file host directory remote_file local_file user pass
 ftp -n $2 > $1 2> $1 <<EOC
 user $6 $7
diff -Nru GXedit1.23/xproc-src/proc.sh GXedit1.23.new/xproc-src/proc.sh
--- GXedit1.23/xproc-src/proc.sh	Tue Feb 23 07:02:43 1999
+++ GXedit1.23.new/xproc-src/proc.sh	Wed Oct 20 23:26:14 1999
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 echo "Welcome to Xproc's console version: proc.sh"
 echo "Hint: You should run this as ./proc.sh|more and to leave, just press ^C"
 menu()
